engines/grim/detection.cpp

```diff
@@ -153,6 +153,21 @@ static const GrimGameDescription gameDescriptions[] = {
 		},
 		GType_GRIM
 	},
+	{
+		// Grim Fandango Hebrew Fan translation (patched)
+		// As English version uses VOX0001.LAB, this needs to be listed first
```

Comment:
I don't understand this comment. This entry is not listed first

Reply:
At first it was, then I saw it doesn't have to be first and forgot to remove the comment, now it's removed